examples/autocorrection.py

```diff
@@ -29,11 +26,11 @@ def _(event):
         """
         b = event.cli.current_buffer
         w = b.document.get_word_before_cursor()
-
-        if w is not None:
-            if w in corrections:
-                b.delete_before_cursor(count=len(w))
-                b.insert_text(corrections[w])
+        if w != '':
+            for word in corrections:
+                if w in word:
```

Comment:
Hi @38elements,

Thank you for this fix! I appreciate it, but I don't think it's entirely correct. The ``w != ''``-part is okay. But this also replaces substrings of the ``corrections`` dictionary by their value. (Doesn't this replace ``lrd`` by ``world``?)